documentation/3_Interoperability.md

```diff
@@ -0,0 +1,375 @@
+applied to Data Spaces.
+
+### Technical
+
+The basis for technical interoperability in Data Spaces is the [Dataspace
+Protocol(DSP)](https://docs.internationaldataspaces.org/ids-knowledgebase/v/dataspace-protocol). This protocol provides a set of specifications
+designed to facilitate interoperable data sharing between entities
+governed by usage control and is based on web technologies. These
+specifications define the schemas and protocols required for entities to
+publish data, negotiate agreements, and access data as part of a
+federation of technical systems that form a Data Space.
+
+The DGA will specify which version(s) of the DSP are mandatory for
+participants of a Data Space. This guarantees that at a technical layer
+participants will be able to interact in a Data Space.
+
+In addition to the DSP, additional protocols, like identity and trust
+protocols can be defined to guarantee minimum technical interoperability
+within a Data Space.
+
+If two Data Spaces mandate their participants to use the same protocols
+it will be make the participation technically easier as participants
+might re-use the same technical components to access data in different
+Data Spaces, instead of having to maintain separate technical system for
+each Data Space.
+
+### Semantic
+
+An important part of interoperability is the semantic models used in a
+Data Space. This includes semantic models of the data that is to be
+shared within the Data Space as well as semantic models for the data that
+describes the Data Space itself (such as policies, participants,
+processes etc).
+
+To successfully participate in a Data Space a participant needs to at
+least understand the semantic model used for policies within the
+Data Space. This is usually pre-defined through the DGA and acceptance of
+the semantic model of the Data Space policies is very often going to be a
+pre-requisite to join the Data Space.
+
+The semantic model of the Data Space's policies helps participants to
+understand the policies that can be negotiated in data sharing
+contracts. Without a common understanding of what individual policies
+mean and the expectations for their execution, it is not possible to
+participate in a Data Space.
+
+The semantic model of the actual data being shared is not a mandatory
+required element, but it greatly enhances the value of the Data Space and
+the data shared within as it enables all participants know what each
+data element means, and how it is constructed. For example, if a field
+refers to "country code", it is necessary that the participants of that
+particular data sharing transaction know how that value is coded (e.g.
+ISO two letter or three letter abbreviation, some numeric
+representation, or whatever else has been chosen); if one participants
+is coding the United Kingdom as "UK" while another uses "GB", the danger
+of misalignment and miscommunication is obvious.
+
+If two DGAs negotiate and agree on the same semantic model for policies
```

Comment:
Wouldn't that be the federation of data spaces?

Reply:
no, federation would also require dependence on common services, and acting on behalf.